SU2_CFD/src/solvers/CDiscAdjSolver.cpp

```diff
@@ -376,7 +371,7 @@ void CDiscAdjSolver::ExtractAdjoint_Solution(CGeometry *geometry, CConfig *confi
 
   if (time_n_needed) {
     SU2_OMP_FOR_STAT(omp_chunk_size)
-    for (auto iPoint = 0u; iPoint < nPoint; iPoint++) {
+    for (auto iPoint = 0ul; iPoint < nPoint; iPoint++) {
```

Comment:
this  iPoint change from 0u to 0ul is less of a necessity then rather a consistency thing